compoments/header.js

```diff
@@ -0,0 +1,322 @@
+import React from 'react';
+
+export default function Header() {
+  return (
+    <>
+      <header
+        className="nsdc-header govuk-header"
+        role="banner"
+        data-module="govuk-header"
+      >
+        <div className="govuk-header__container govuk-width-container">
+          <div className="govuk-header__logo">
+            <a
```

Comment:
Can we add an aria-label to this link.

`aria-label="Newark and Sheerwood Council Home Page"` for example.